.github/workflows/canary-integration-test.yml

```diff
@@ -1421,7 +1424,7 @@ jobs:
       - name: set up tmate session for debugging
         if: failure() && env.USE_TMATE
         uses: mxschmitt/action-tmate@v3
-        timeout-minutes: 60
+        timeout-minutes: 600
```

Comment:
revert this since not required.

Reply:
So it is been reverted in the second commit

Reply:
Yes right